docs/getting-started/install-guide.md

```diff
@@ -132,4 +132,28 @@ Note: Windows 7/8/10 natively provides support for `.zip` files, therefore it is
 
 If you decide to download the `.tar.gz` file, it will require two separate extractions in some third-party application (7-zip, winRAR, etc..) to get to the actual binaries.
 
------------------------
+---
+
+## Decrediton
+
+Decrediton is released with the Binary Releases and can be found here: [https://github.com/decred/decred-binaries/releases](https://github.com/decred/decred-binaries/releases). As of v0.8.2, Decrediton is only available for Linux and Mac and is technically a beta release. There are known bugs in the software that are being worked on.
+
+> macOS/OSX
+
+1. Download the `decrediton-0.8.2.dmg` file.
+
+2. Double click the `decrediton-0.8.2.dmg` file once downloaded to mount the disk image.
+
+3. Drag the decrediton.app into your Applications folder within the disk image.
```

Comment:
Technically is it a link to the applications folder.  So maybe you should say: 'into the link to your Applications folder within...'